### DISCUSSION TITLE
Config Showcase

### DISCUSSION THREAD

**Discussion by WillPower3309** [WillPower3309/swayfx#138] [Show and tell]
A place to share all of your desktop screenshots! Please include links to configs :)

**Comment by iamkarlson** [comment]
my [dotfiles](https://github.com/iamkarlson/dotfiles/tree/feature/thinkpad) with swayfx+waybar+sworkstyle+swaync. _Link points towards feature/thinkpad branch as swayfx does work with nvidia and that's my main setup._ 

![image](https://user-images.githubusercontent.com/4534374/228936323-88cabbd7-78fb-4a25-8810-bd0a99218b8e.png)

**Reply by ErikReider** [reply]
💪

**Comment by head-gardener** [comment]
![20230525_12h56m06s_grim](https://github.com/WillPower3309/swayfx/assets/49650767/09e506d9-9dbe-4480-9ad7-dd210b7e491b)
[link](https://github.com/head-gardener/dotfiles)
Just wanted to say thanks for the great work done on blur! I will come back shortly to finishing my PR's once I sort out school assignments.

**Reply by Doomer-Cat** [reply]
lain nice

**Comment by sleeplessKomodo** [comment]
https://github.com/user-attachments/assets/0b6c7b95-f104-4ad6-aea5-eab4593561d3

I just installed arch last night, so there's still some kinks for me to work out

I am working on setting up a self hosted Forgejo instance, where I will host my dotfiles, so I will update this post then. 

My main waybar is on my 2nd display, because I have two displays stacked vertically. 

(this is my light theme, I use dracula at night)
